### PR TITLE
Document how to pass parts of release announcement to Quarkus team

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc [hunk 1]
```diff
@@ -210,7 +210,10 @@ cd generated-platform-project/quarkus-camel/integration-tests
 mvn clean test
 ----
 +
-* If all tests are passing, send a pull request to the Platform
+* If all tests are passing, send a pull request to the Platform.
+  If there are some new features, fixes, etc. in the release that would be worth to mention in the upcoming Quarkus announcement,
+  you can tag the pull request with `release/noteworthy-feature` label and provide a short description of those features.
```
Comment (zbendhiba) [inline docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:215]: the description of features in the description of PR ?

**Review by zbendhiba — APPROVED** [review]

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc (hunk 1 above)
Comment (ppalaga) [reply docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:215]: Yes, let me state it more clearly.

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc [hunk 2]
```diff
@@ -210,7 +210,10 @@ cd generated-platform-project/quarkus-camel/integration-tests
 mvn clean test
 ----
 +
-* If all tests are passing, send a pull request to the Platform
+* If all tests are passing, send a pull request to the Platform.
+  If there are some new features, fixes, etc. in the release that would be worth to mention in the upcoming Quarkus announcement,
+  you can tag the pull request with `release/noteworthy-feature` label and provide a short text about those features in the PR description.
```
Comment (ppalaga) [inline docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:215]: @zbendhiba better?

Comment (zbendhiba) [reply docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:215]: @ppalaga Yes, thanks.

**Review by aldettinger — APPROVED** [review]

**Pull request merged**